feat(profiling): add browser profiling docs (#7468)
getsentry/sentry-docs: src/wizard/javascript/profiling-onboarding/javascript/1.install.md

---
name: Browser (JS Self Profiling)
doc_link: https://docs.sentry.io/platforms/javascript/profiling/
support_level: beta
type: framework
---

> #### Install

Reviewer: ```suggestion
## Install
```

> Our browser profiling integration is built upon the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it is therefor in beta stage and is likely to only to move out as the official spec progresses. There are obviously risks with using a beta package - we've made sure that the SDK gracefully degrades in unsupported environments.

Reviewer: ```suggestion
Because our browser profiling integration is built on top of the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it's in beta and will likely only move out once the official spec progresses. There are some risks involved with using a beta package. We've made sure that the SDK gracefully degrades in unsupported environments.
```

> Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic.

Reviewer 2: We could also link to this status page which shows whether other browsers are adopting it: https://chromestatus.com/feature/5170190448852992

> Please note that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherintly be biased towards that demographic - this is something you need to be aware of when making decisions.

Reviewer: ```suggestion
Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic.
```

> Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic - this is something you need consider if you are basing your decisions on the data collected. We hope that as the API gains adoption, other browsers will implement it as well. If you find browser profiling feature helpful and would like to see it gain further adoption, please consider supporting the spec at the official [WICG repository](https://github.com/WICG/js-self-profiling).

Reviewer: ```suggestion
Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic. This is something you'll need to consider if you're basing your decisions on the data collected. We hope that as the API gains adoption, other browsers will implement it as well. If you find browser profiling feature helpful and would like to see it gain further adoption, please consider supporting the spec at the official [WICG repository](https://github.com/WICG/js-self-profiling).
```

> Install our JavaScript browser SDK using either `yarn` or `npm`, the minimum version that supports profiling is **7.60.0**

Reviewer: ```suggestion
Install our JavaScript browser SDK using either `yarn` or `npm`, the minimum version that supports profiling is **7.60.0**.
```
